src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.server.ts

```diff
@@ -1,47 +1,69 @@
 import gql from "graphql-tag";
 import { print as printQuery } from "graphql";
+import { error } from "@sveltejs/kit";
 import parishTopoJSON from "./parishes.json";
 
-import type { FireWeatherMapQuery } from "./$queries.generated";
+import type { FireWeatherMapPageQuery } from "./$queries.generated";
 
 const query = gql`
-  query FireWeatherMap {
-    fireDangerMapCollection(limit: 1) {
+  query FireWeatherMapPage($metadataID: String!, $preview: Boolean = false) {
+    fireDangerMapCollection(
+      where: { pageMetadata: { sys: { id: $metadataID } } }
+      limit: 1
+      preview: $preview
+    ) {
       items {
+        title
         description {
           json
         }
+        pageMetadata {
+          sys {
+            id
+          }
+        }
       }
     }
   }
 `;
 
-export const load = async ({ locals: { getKVClient }, locals: { contentfulClient } }) => {
-  try {
-    const kvClient = await getKVClient();
-    const fireWeatherData = await kvClient.getFireWeatherData();
-
-    if (fireWeatherData === undefined) {
-      throw new Error("Error fetching fire weather data.");
-    }
+export const load = async ({ parent, locals: { getKVClient, contentfulClient } }) => {
+  fetchData: {
+    if (!contentfulClient) break fetchData;
+    const { pageMetadataMap, pathsToIDs } = await parent();
+    const metadataID = pathsToIDs.get("/land/fire/safety/fire-danger-map");
+    if (!metadataID) break fetchData;
+    const pageMetadata = pageMetadataMap.get(metadataID);
+    if (!pageMetadata) break fetchData;
```

Comment:
Added in the good ol' `fetchData` label so that we can more easily return a 404 in the case of content errors (as opposed to map data errors).

The other benefit here is that this gives us the `pageMetadata` entry so that we can update the meta tags and such ([in the base layout](https://github.com/la-ldaf/ldaf-site/blob/9aa03aa9241b5f32bad23bbb8245454944269393/src/routes/%2Blayout.svelte#L55)).

Reply:
good catch, I forgot about some of those other tie-ins to `pageMetadata`